### PR TITLE
feat(frontend): enhance dapp modal

### PR DESCRIPTION
**Pull request by loki344**
# Motivation

Integrate testing feedback: More social links, styling issue of image on small screens.

# Changes

- Integrate new social links with icons.
- Fix Image height

before: 
image

after: 
image

### CODE REVIEW

**Comment by peterpeterparker** [comment]
Does it still looks good if the title and url are longer than in the provided screenshots?

379193860-518ee862-15bd-4eba-845a-cdb2cb4beb98

File: src/frontend/src/lib/components/ui/ImgBanner.svelte [hunk 1]
```diff
@@ -4,11 +4,13 @@
...
 	export let width: string | undefined = undefined;
+	export let height: string | undefined = undefined;
+	export let fitHeight = false;
 	export let size: 'small' | 'big' = 'big';
 </script>
 
 <div class="relative block" class:h-56={size === 'big'} class:h-40={size === 'small'}>
-	<div class="absolute w-full">
-		<Img {src} {alt} {width} />
+	<div class="absolute h-full w-full">
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/ui/ImgBanner.svelte:13]: Did you double checked that if the network is slow the change does not result in elements shift in the UI? Asking just to be sure everything remains still in place.

**Comment by loki344** [comment]
> Does it still looks good if the title and url are longer than in the provided screenshots?

yup, added a flex wrap for this: 
image

File: src/frontend/src/lib/components/ui/ImgBanner.svelte (hunk 1 above)
Comment (loki344) [reply src/frontend/src/lib/components/ui/ImgBanner.svelte:13]: yup

Uploading Bildschirmaufnahme 2024-10-23 um 14.46.20.mov…

**Pull request merged**